### ISSUE TITLE
Dark Mode and sass changes

### ISSUE THREAD

**Issue by AZaletski24** [Tvick22/platformer3x#6]
### Dark Mode

- [ ] Identify key sass files related to creating a dark mode
- [ ] Create a button that will toggle dark/light mode

### Sass Changes

- [ ] Change leaderboard styles to match by level
- [ ] Create customization options that change the whole game

**Comment by myamadriaga** [comment]
# Planning/Idealization for the Multiple-Themed Game-Mode 

## On Tuesday, we idealized a button choice/feature on the game (most likely within settings) which will change the theme of the game page/game, such as changing to light mode or back to dark mode. 

For now, we are creating more ideas as to how we can construct the code for 
- changing of themes, 
-  adding the button in settings 
-  creating a function that allows a theme change 
-  adding multiple theme functions (which may mean multiple scss files) 

After looking into the '''html dark-mode.scss''' file, I understand that I would need to create multiple scss files in order to connect it to desired themes. 
while I was experimenting, I just created a light-mode.scss file, and used similar keywords and paths as the dark-mode file. 
EX:

```sass
:root {--default-canvas-filter: invert(100%);}
$high-emph: rgba(white, 0.87);
$med-emph: rgba(white, 0.69); 
$low-emph: rgba(white, 0.38);
$dark-grey : #121212; 
// $dark-grey : #1F1B24;
$overlay: mix($dark-grey, white, 95%);
$overlay-light: mix($dark-grey, white, 86%);
```
I got rid of the canvas filter invert, as well as the hex codes that called black, or dark grey colors. 

Looking into '''js SettingsControl.js,''' I saw code like:
```js 
  }
    get multiplayerButton() {
        const div = document.createElement("div");
        div.innerHTML = "Multiplayer: "; // label
    
        const button = document.createElement("button"); // button for Multiplayer
        button.innerText = String(Socket.shouldBeSynced);
    
        button.addEventListener("click", () => {
            // dispatch event to update difficulty
            button.innerText = String(Socket.changeStatus());
        });
```
There, I understand that I will have to use a good amount of this code/function to create another button that corresponds to the theme changing button. There will be radio buttons, that correspond to each changeTheme function. 

(similar to this:)
```js
 <div id="controls"> <!--basic radio buttons which can be used to check whether each individual animaiton works -->
            <input type="radio" name="animation" id="punch 1" checked>
            <label for="punch 1">punch 1</label><br>
            <input type="radio" name="animation" id="punch 2">
            <label for="punch 2">punch 2</label><br>
            <input type="radio" name="animation" id="punch 3">
```

The main JS file that calls the changeTheme function, will be a base for what everything uses. I assume there should be a different JS file that corresponds to what each theme is as well. What it also could be like is:
- creating a function that states on press, it calls on the indicated scss file, then will call on the main piece of code (changeTheme) --> indicating a change of theme on the game/website. 

Moreover, I also saw functions in the code like this: 
```js
get difficultyInput() {
        const div = document.createElement("div");
        div.innerHTML = "Difficulty: "; // label
    
        const difficulty = document.createElement("select"); // dropdown for difficulty
        const options = ["Easy", "Normal", "Hard", "Impossible"];
    
        options.forEach(option => {
            const opt = document.createElement("option");
            opt.value = option.toLowerCase();
            opt.text = option;
            difficulty.add(opt);
        });
```
basically where it changes the input, and its seeming to offer the choices for the change of input. I think this code might be beneficial for the ideal theme change function in the game. 

I don't currently have results that work for the game, so there are no images. 

## Plans when done with task: 
- maybe the theme changes according to the level
- fixing bugs after changing/finalizing the repo changes.
